docs/csharp/language-reference/operators/switch.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "switch expressions - C# reference"
```

Comment:
All other articles are in single. "default value expressions" is an exception because there are two kinds of that.
```suggestion
title: "switch expression - C# reference"
```